### ISSUE TITLE
Add references to documentation

### ISSUE THREAD

**Issue by brunaw** [zeileis/distributions3#17]
Should we have a `.bib` file to put the references used?

**Comment by alexpghayes** [comment]
I'd prefer to put references directly in the function documentation, so that there isn't an extra step for users to find them. Ideally we should include a URL to a copy of the reference in the function documentation as well.

**Comment by brunaw** [comment]
Absolutely! but I guess the references for each documentation could be created with a unified `.bib` file that we just use to cite?

**Comment by alexpghayes** [comment]
Oh I did know this was possible. Can you shoot me a link to an example /
documentation on how this works?

**Comment by brunaw** [comment]
I actually never did it myself :p but I remember seeing it a few times. I found this tutorial which might be exactly what we want: https://cran.r-project.org/web/packages/Rdpack/vignettes/Inserting_bibtex_references.pdf

**Comment by alexpghayes** [comment]
I may do this down the line if I teach with `distributions3`, but otherwise this probably won't happen unless someone is excited about it.
